roles/lvm/templates/lvm.conf.j2

```diff
@@ -0,0 +1,31 @@
+{% set used_lvm_devices = [] %}
+{% set cinder_lvm_devices_filter_override = [] %}
+{% if cinder_lvm_devices_filter_override|length > 0 %}
+    {% set used_lvm_devices = cinder_lvm_devices_filter_override %}
+{% else %}
+    {% set lv_devices = lvm_devices.stdout.split('\n') %}
+    {% if lv_devices|length > 0 %}
+        {% for net in lv_devices %}
+            {% if net != '' %}
+                {% set lv_device = '"a/' + net + '/"' %}
+                {% if used_lvm_devices.append(lv_device) %}{% endif %}
+            {% endif %}
+        {% endfor %}
+    {% endif %}
+
+# Ansible Discovered LVM Devices {{ lv_devices }}
+{% endif %}
+
+{% if used_lvm_devices|length <= 0 %}
+    {# If there are no LVM devices present, allow all devices to be scanned #}
+    {% if used_lvm_devices.append('"a/.*/"') %}{% endif %}
```

Comment:
We actually want to not allow anything, because if there is no LVMs, we will allow plugged volumes to be scanned, this is how I fixed it in an environment where the base OS is not using LVM:

```
        filter = [ "r|.*|" ]
```